testing/set-env-vars.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -x
+
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION=1.25.0
+
+LOCALBIN=`pwd`/bin
+ENVTEST=${LOCALBIN}/setup-envtest
+
+KUBEBUILDER_ASSETS="$(${ENVTEST} use ${ENVTEST_K8S_VERSION} --bin-dir ${LOCALBIN} -p path)"
+
+# do not existing cluster by default
+USE_EXISTING_CLUSTER=${USE_EXISTING_CLUSTER:="false"}
+
+# setting verbose output for ginkgo
+# for very verbose set "ginkgo.vv"
+GINKGO_VERBOSE_FLAG=${GINKGO_VERBOSE_FLAG:="ginkgo.v"}
+
+# GINKGO_LABELS="provisioning,test-update"
+GINKGO_LABEL_FILTER=${GINKGO_LABEL_FILTER:=""}
```

Comment:
how those two relate each other? GINKO_LABELS is not exported

Reply:
Invalid name in the comment. Corrected. The exported name is GINKGO_LABEL_FILTER.